Update fastqc dependency to version 0.11.4

gregvonkuster opened the pull request.

bgruening commented on tools/fastqc/rgFastQC.xml:
```diff
@@ -1,7 +1,7 @@
-<tool id="fastqc" name="FastQC" version="0.63">
```
The version number should never be smaller than the old one.

gregvonkuster replied:
Yikes, sorry!  Just a sec...

gregvonkuster replied:
Better now.  ;)

bgruening replied:
jupp!

bgruening commented:
:+1:

gregvonkuster commented:
@bgruening @martenson 
If I'm missing something here, please let me know - my mind doesn't work as well as it used to... ;)

It looks like an upload of this tool to the Main Tool Shed back in 2014 deleted everything but the tool_dependencies.xml file?  Can we get the tool that I recently updated to use version 0.11.4 of fastqc re-uploaded to this repo?

Here is what is currently in the repo:
Contents:
fastqc
tool_dependencies.xml

Here is the changeset that deleted everything:

Repository 'fastqc'
Clone this repository:
hg clone https://toolshed.g2.bx.psu.edu/repos/iuc/fastqc
Changeset 1:39b1c10532a4 (2014-01-18)
Previous changeset 0:a1472b1151d4 (2014-01-18) Next changeset 2:2611a96c30b7 (2014-01-18)
Commit message: 
Remove intermediate directory
added: 
rgFastQC.py 
rgFastQC.xml 
test-data/1000gsample.fastq 
test-data/fastqc_contaminants.txt 
test-data/fastqc_report.html 
test-data/gentestdata.sh 
tool_dependencies.xml
removed: 
fastqc/rgFastQC.py 
fastqc/rgFastQC.xml 
fastqc/test-data/1000gsample.fastq 
fastqc/test-data/fastqc_contaminants.txt 
fastqc/test-data/fastqc_report.html 
fastqc/test-data/gentestdata.sh 
fastqc/tool_dependencies.xml

nsoranzo commented:
@gregvonkuster I think you are looking at https://toolshed.g2.bx.psu.edu/view/iuc/fastqc/ which is from IUC not devteam. This package has been updated when you created `galaxyproject_tools-iuc/packages/package_fastqc_0_11_4/.shed.yml` which unfortunately contains a typo: `name: fastqc` should be `name: package_fastqc_0_11_4`.

I will update the `.shed.yml` and deprecate the MTS package again.

gregvonkuster commented:
Sorry for the screw up, but thanks for the fix @nsoranzo!

gregvonkuster commented:
Can this tool https://toolshed.g2.bx.psu.edu/view/devteam/fastqc/2d094334f61e now be updated in the MTS with the upgrade in this PR?
